java/vector/src/test/java/org/apache/arrow/vector/table/RowTest.java

```diff
@@ -1,856 +0,0 @@
-/*
```

Comment:
I'll put this back before I un-draft the PR. I could not get it to compile locally, so I deleted it for now.